cogs/annual_handover_and_reset.py

```diff
@@ -222,3 +227,113 @@ async def annual_roles_reset(self, ctx: TeXBotApplicationContext) -> None:
 
         logger.debug("Execution of reset roles command complete!")
         await initial_response.edit(content=":white_check_mark: Role reset complete!")
+
+class AnnualYearChannelsIncrementCommandCog(TeXBotBaseCog):
+    """Cog class that defines the "/increment-year-channels" command."""
+
+    @discord.slash_command(  # type: ignore[no-untyped-call, misc]
+        name="increment-year-channels",
+        description="Iterates the year channels, archiving and creating channels as needed.",
```

Comment:
Use new command name in description: `description="Increments the year channels, archiving and creating channels as needed.`